src/mmw/bundle.sh

```diff
@@ -113,6 +118,7 @@ done
 VAGRANT_COMMAND="cd /opt/app && \
     $ENSURE_DIRS_EXIST && { \
     $COPY_IMAGES_COMMAND &
+    $COPY_SIMPLE_APP_DATA_COMMAND &
```

Comment:
We actually don't need any changes to `bundle.sh`. Because `model.csv` lives under the `static` directory in a Django app, it is automatically packaged and deployed by the `collectstatic` command:

```bash
./scripts/bundle.sh collectstatic --noinput
```

This command is run during provisioning, thus we don't need the additional commands in `bundle.sh`.